python/looker_sdk/rtl/api_settings.py

```diff
@@ -114,10 +116,19 @@ def read_ini(filename: str, section: Optional[str] = None) -> Dict[str, str]:
             # If section is not specified, use first section in file
             section = section or cfg_parser.sections()[0]
             config_data = dict(cfg_parser[section])
-            # If setting is an empty string, remove it
-            for setting in list(config_data):
-                if config_data[setting] in ['""', "''", ""]:
-                    config_data.pop(setting)
             config_data["_section"] = cast(str, section)
             config_data["_filename"] = cast(str, filename)
         return config_data
+
+    @classmethod
+    def _clean_input(cls, config_data: Dict[str, str]) -> Dict[str, str]:
+        for setting, value in list(config_data.items()):
+            # Remove empty setting values
+            if not isinstance(value, str):
+                continue
+            if value in ['""', "''", ""]:
+                config_data.pop(setting)
+            # Strip quotes from setting values
+            elif value.startswith(('"', "'")) or value.endswith(('"', "'")):
+                config_data[setting] = value.strip("\"'")
```

Comment:
You only want to strip beginning and ending quote values. Does `value.strip()` remove all occurrences?

Reply:
It only removes leading/trailing occurrences: https://docs.python.org/3/library/stdtypes.html#str.strip

Reply:
Ah. Most languages I know call that `trim` of some sort. Thanks.